src/Microsoft.IdentityModel.Clients.ActiveDirectory/IAdalLogCallback.cs

```diff
@@ -57,8 +58,20 @@ public enum LogLevel
     }
 
     /// <summary>
-    /// Callback for capturing ADAL logs to custom logging schemes.
+    /// Callback delegate that allows the developer to consume logs handle them in a custom manner.
     /// </summary>
+    /// <param name="level">Log level of the message</param>
+    /// <param name="message">Pre-formatted log message</param>
+    /// <param name="containsPii">Indicates if the log message contains PII. If Logger.PiiLoggingEnabled is set to 
+    /// false then this value is always false.</param>
+    public delegate void LogCallback(LogLevel level, string message, bool containsPii);
```

Comment:
This is an addition to the public API (and therefore we need to increase the release minor number) => 3.18 and not 3.17.4

Reply:
done.